packages/react/src/components/pickers/Suggestions/SuggestionsController.ts

```diff
@@ -127,7 +127,7 @@ export class SuggestionsController<T> {
       return {
         item: suggestion,
         selected: false,
-        ariaLabel: (<any>suggestion).name || (<any>suggestion).primaryText,
+        ariaLabel: (<any>suggestion).ariaLabel || (<any>suggestion).name || (<any>suggestion).primaryText,
```

Comment:
I'd actually be somewhat inclined to just set this to `suggestion.ariaLabel`, since even with `||` or `??`, it'll still not be possible to remove the `aria-label` with `ariaLabel: null`. In almost all cases, I think we probably should not set the `aria-label` at all, and allow the option to get its name directly from its text content.
```suggestion
        ariaLabel: (<any>suggestion).ariaLabel,
```

Reply:
Sounds good to me! Works better for our hosting scenario actually with that update. :P I agree probably in most all cases screenreader should just read all its child text content to get the full suggestion information. And then yeah if hosts dont want that experience they'd either hide any secondary text or override with this prop. Good idea!